docs/views/tutorials-and-examples.html

```diff
@@ -176,7 +176,7 @@ <h2 class="govuk-heading-m">Page templates</h2>
         </li>
         <li>
           <a href="/docs/templates/check-your-answers">
```

Comment:
this link needs to change to the new file name

Reply:
```suggestion
          <a href="/docs/templates/check-answers">
```

Reply:
Thanks @joelanman, that's good to know for future reference!